ui/page/about_page.go

```diff
@@ -85,24 +85,51 @@ func (pg *AboutPage) Layout(gtx layout.Context) layout.Dimensions {
 	return components.UniformPadding(gtx, body)
 }
 
+var inset = layout.Inset{
+	Top:    values.MarginPadding20,
+	Bottom: values.MarginPadding20,
+	Left:   values.MarginPadding16,
+	Right:  values.MarginPadding16,
+}
+
+func containerPadding() components.Container {
+	return components.Container{
+		Padding: inset,
+	}
+}
+
+func padding() layout.Inset {
+	return inset
+}
+
 func (pg *AboutPage) layoutRows(gtx layout.Context) layout.Dimensions {
 	w := []func(gtx C) D{
 		func(gtx C) D {
-			return components.EndToEndRow(gtx, pg.version.Layout, pg.versionValue.Layout)
+			return containerPadding().Layout(gtx, func(gtx C) D {
+				return components.EndToEndRow(gtx, pg.version.Layout, pg.versionValue.Layout)
+			})
```

Comment:
Rather than create a method for container, you could use Container from the components package like so

```suggestion
			return components.Container{Padding: in}.Layout(gtx, func(gtx C) D {
				return components.EndToEndRow(gtx, pg.version.Layout, pg.versionValue.Layout)
			})
```

Where the variable `in` is a variable to the inset defined within the method `layoutRows()` as in
```go 
       in := layout.Inset{
		Top:    values.MarginPadding20,
		Bottom: values.MarginPadding20,
		Left:   values.MarginPadding16,
		Right:  values.MarginPadding16,
	}
```